### PR TITLE
Extend WebGPU origin trial

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR updates the WebGPU documentation to mention the extension of the WebGPU origin trial.

Live preview: https://deploy-preview-3274--developer-chrome-com.netlify.app/docs/web-platform/webgpu/#enabling-support-during-the-origin-trial-phase

@Kangz Please review.

### CODE REVIEW

File: site/en/docs/web-platform/webgpu/index.md [hunk 1]
```diff
@@ -73,7 +73,12 @@ To experiment with WebGPU locally, without an origin trial token, enable the
 ### Enabling support during the origin trial phase
 
 Starting in Chrome&nbsp;94, WebGPU is available as an origin trial in Chrome. The
-origin trial is expected to end in Chrome&nbsp;105 (Sep 21, 2022).
+origin trial is expected to end in Chrome&nbsp;109 (Feb 1, 2023).
+
+{% Aside 'caution' %}
+The origin trial is expected to [pause for 2 weeks](https://groups.google.com/a/chromium.org/g/blink-dev/c/gBMsYQ1Aurw/m/xl1znc7fAAAJ)
+starting in Chrome&nbsp;106 (Sep 20, 2022).
```
Comment (Kangz) [inline site/en/docs/web-platform/webgpu/index.md:80]: sep 27th?

**Review by Kangz — COMMENTED** [review]
LGTM except the small typo in the date

File: site/en/docs/web-platform/webgpu/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply site/en/docs/web-platform/webgpu/index.md]: Thanks!

**Comment by beaufortfrancois** [comment]
Thank you @Kangz!

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**